rumqttd/src/server/broker.rs

```diff
@@ -219,19 +213,13 @@ impl Broker {
         if let Some(ws_config) = &self.config.ws {
             for (_, config) in ws_config.clone() {
                 let server_thread = thread::Builder::new().name(config.name.clone());
-                let server = Server::new(
-                    config,
-                    self.router_tx.clone(),
-                    Ws {
-                        codec: MessageCodec::server(),
-                    },
-                );
+                let server = Server::new(config, self.router_tx.clone(), V4);
```

Comment:
Another option could be to have it as a property on ServerSettings. This would deviate from how v4 & v5 are specified for the normal connection (If you don't move those aswell, but might be out of scope for this PR). Let me know which you would prefer 👍🏼

Reply:
I agree with you that this might get out of scope for this PR. For now, lets just add a `TODO` there, mentioning something like we want to be able to configure protocol to use.